src/sentry/conf/server.py

```diff
@@ -1132,7 +1139,7 @@ def create_partitioned_queues(name):
 )
 
 # Delay (in ms) to induce on API responses
-SENTRY_API_RESPONSE_DELAY = 0
+SENTRY_API_RESPONSE_DELAY = 150 if IS_DEV else None
```

Comment:
Definitely not a fan of literally everyone who is running sentry on macOS getting 150ms delay. I assume this will also affect tests.

I'd rather we spit out a sentry.conf.py with some of these things set automatically when you do `sentry init --dev` which already spits out a different config file. Rather than making these hard assumptions all the time.